build/DownloadCFamilyPlugin/DownloadCFamilyPlugin.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31112.23
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DownloadCFamilyPlugin", "DownloadCFamilyPlugin.csproj", "{37E4D27F-FDAB-4D1D-8269-362ADC06D777}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{37E4D27F-FDAB-4D1D-8269-362ADC06D777}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{37E4D27F-FDAB-4D1D-8269-362ADC06D777}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{37E4D27F-FDAB-4D1D-8269-362ADC06D777}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{37E4D27F-FDAB-4D1D-8269-362ADC06D777}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {12B97225-FE66-4177-8E41-2D51616C13DB}
+	EndGlobalSection
+EndGlobal
```

Comment:
VS kept nagging me to save this file, so in the end I saved it and added it to source control